students/sungho-moon/user/views.py

```diff
@@ -1 +1,33 @@
+import json
+
+from django.http  import JsonResponse
+from django.views import View
+
+from user.models  import Signup
+
+class SignupView(View):
+
+    def post(self,request):
+        data    = json.loads(request.body)
+        
+        try:
+            if (Signup.objects.filter(nick_name=data['nick_name']).exists()) or (Signup.objects.filter(email=data['email']).exists()) or (Signup.objects.filter(phone_number = data['phone_number']).exists()):
```

Comment:
```suggestion
            if (Signup.objects.filter(nick_name=data['nick_name']).exists()) or 
                (Signup.objects.filter(email=data['email']).exists()) or 
                (Signup.objects.filter(phone_number = data['phone_number']).exists()):
```
이러한 편이 조금 더 가독성이 좋아 보입니다.